app/models/user.rb

```diff
@@ -36,4 +36,9 @@ def self.calculate_win_ratio(won, finished)
     return 0 unless finished > 0
     (won.to_f / finished * 100).round(2)
   end
+
+  def unranked?
+    return true if results.size < 5
```

Comment:
Can there be unfinished results? Would `return true if games_finished < 5` be better?